### DISCUSSION TITLE
Is it possible to install an msi file?

### DISCUSSION THREAD

**Discussion by LinuxToasty** [ruanformigoni/gameimage#24] [Q&A]
Hi there, 

If I have an windows installer in a msi format that I have added to the rom directory it doesn't work. What is the best way round this?

Regards

**Comment by ruanformigoni** [comment]
Hello! Have you tried to install it during the configuration phase with 'wine /path/to/msi/exe' ?

**Comment by LinuxToasty** [comment]
Thank you for taking the time to answer.

I have tried but the installer doesn't launch at all. 
I have also tried wine msiexec /i  and the path to the msi installer and also wine start but nothing seems to bring the installer up.

I have tried the installer in standard wine and this works as it should.

Thanks again.

**Comment by ruanformigoni** [comment]
No problem, could you try to install it through the interface that opens when you run `wine uninstaller`? If that doesn't work, I suggest as a workaround, to set the `WINEPREFIX` variable to the prefix that is inside the build directory, and then use the wine version that works (non-appimage) to install the program.

**Comment by ruanformigoni** [comment]
I also have [this](https://mega.nz/file/9wYXwLrR#DwujwGXKY29PAYaljrxegNsMBCnCYVR2sBV_0VW5wDw) version of wine, that uses a difference packing method than appimage, if you would like to try it out.

**Comment by LinuxToasty** [comment]
Thank you for your help.
I tried the wine uninstaller but this could not find my msi file or anything in my filesystem at all.
I tried the version of wine that you uploaded but unfortunately I have glibc errors as I am running Debian stable with an older version of the libraries.
I did manage to resolve by using the system wine to install into the prefix so that worked!
A huge thank you for taking the time and effort to help.

**Reply by ruanformigoni** [reply]
Glad it worked out! :smile:

**Comment by ruanformigoni** [comment]
Hey there, forgot to ask, is the `.msi` file that you were trying to install publicly available? I'd like to investigate to see if I can fix this issue. Thanks!

**Comment by LinuxToasty** [comment]
Hi there,
I tried a few to be honest. Including https://github.com/LoneWolfHT/minetest-installers-windows to test.
Minetest is free and open source so would probably the best one to look at.
